docs/contrib/PlatformBringUp.md

```diff
@@ -406,10 +423,8 @@ To validate the new platform, simply run a design through the flow using the new
 
 # Authors/Contributors
 
-
-
 * James Stine - Oklahoma State University
-* Teo Ene
-* Ricardo Hernandez
-* Ryan Ridley
-* Indira Iyer
+* Teo Ene - Oklahoma State University
+* Ricardo Hernandez - Oklahoma State University
+* Ryan Ridley - Oklahoma State University
+* Indira Iyer - TODO
```

Comment:
TODO?

Reply:
I was going to leave this for @dralabeing to add for herself since I wasn't sure what to put